src/main/java/com/internship/training/StreamsChallenge.java

```diff
@@ -5,11 +5,12 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.List;
+import java.util.*;
```

Comment:
Because it occurs for import collisions and alters namespace, and the problem it generates is that the code does not compile and is less maintainable.

Reply:
It can generate some problems during compilation or if you have classes with the same name.